packages/jest-message-util/src/__tests__/messages.test.ts

```diff
@@ -27,63 +28,76 @@ const unixStackTrace =
   at Spec.addResult (../jest-jasmine2/build/jasmine-2.4.1.js:333:34)
   at Expectation.addResult (../jest-jasmine2/build/jasmine-2.4.1.js:591:21)
   at Expectation.toBe (../jest-jasmine2/build/jasmine-2.4.1.js:1504:12)
-  at Object.it (build/__tests__/messages-test.js:45:41)
+  at Object.it (/build/__tests__/messages-test.js:45:41)
```

Comment:
`stack-tools` is a lot stricter than `stack-utils` was. As this test demonstrates certain changes are definitely breaking, though as far as I know they only break usages that do not occur in ~~the real world~~ modern versions of node.